util/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "915336943764",
+    "project_id": "triton-android-sdk",
+    "storage_bucket": "triton-android-sdk.appspot.com"
+  },
```

Comment:
Why adding this to the public repo? This should be something private IMO

Reply:
I agree this should be in here.